addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -572,9 +567,9 @@ def action_refuse(self):
         if any(holiday.state not in ['confirm', 'validate', 'validate1'] for holiday in self):
             raise UserError(_('Allocation request must be confirmed or validated in order to refuse it.'))
 
-        validated_holidays = self.filtered(lambda hol: hol.state == 'validate1')
-        validated_holidays.write({'state': 'refuse', 'first_approver_id': current_employee.id})
-        (self - validated_holidays).write({'state': 'refuse', 'second_approver_id': current_employee.id})
+        # validated_holidays = self.filtered(lambda hol: hol.state == 'validate1')
+        # validated_holidays.write({'state': 'refuse', 'approver_id': current_employee.id})
```

Comment:
Is this code commented because it was leading to an error, or is it really supposed to be removed ?